install/tools/setupIstioVM.sh

```diff
@@ -52,6 +52,7 @@ function istioNetworkInit() {
 # Install istio components and certificates. The admin (directly or using tools like ansible)
 # will generate and copy the files and install the packages on each machine.
 function istioInstall() {
+  sudo rm "/usr/local/bin/node-agent"
```

Comment:
a) don't need sudo 
b) why would this be needed ?

Reply:
You are right. I removed this line in latest commit.